### DISCUSSION TITLE
How to avoid upgrading Ableton Live from 11 to 12?

### DISCUSSION THREAD

**Discussion by FlavioB79** [Homebrew/discussions#5191] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.11
ORIGIN: https://github.com/Homebrew/brew
HEAD: d7d4c8266210c024b93a450a7d357cec0b46a1bb
Last commit: 6 days ago
Core tap JSON: 10 Mar 15:38 UTC
Core cask tap JSON: 10 Mar 15:38 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hi.
Ableton Live has now received an upgrade - a major release upgrade: the new branch is version 12, but a user can't simply upgrade from 11 to 12. It has to buy the upgrade license.
So my question is: how may I still get upgrades on the v11 branch of Ableton Live and ignore v12? Today I ran "brew upgrade --greedy" and it started downloading Ableton Live 12 - which I had to interrupt immediately.
Thanks,
F.

**Comment by miccal** [comment]
`ableton-live-suite11` was added to `homebrew-cask-versions` here: https://github.com/Homebrew/homebrew-cask-versions/pull/19571

So you can `brew uninstall ableton-live-suite` and then `brew install ableton-live-suite11` to continue to receive updates for version 11.

**Comment by FlavioB79** [comment]
Hi Miccal.
Well, your suggestion didn't go well. I've uninstalled Ableton Live Suite
11 and now can't install it with your command.
This is the output I get:

brew install ableton-live-suite11

Warning: No available formula with the name "ableton-live-suite11".

==> *Searching for similarly named formulae and casks...*

==> *Casks*

ableton-live-lite
ableton-live-suite


To install ableton-live-lite, run:

  brew install --cask ableton-live-lite


Can you please help? I'm stuck without Ableton Live Suite 11 now.

BR,
Flavio.


https://www.instagram.com/boniforti_music
https://soundcloud.com/boniforti_music
https://bonny-j.bandcamp.com


Am Mo., 11. März 2024 um 02:02 Uhr schrieb Miccal Matthews <
***@***.***>:

**Comment by daeho-ro** [comment]
Tap first,
```
brew tap homebrew/cask-versions
brew search homebrew/cask-versions/ableton-live-suite11
==> Casks
ableton-live-suite11
```
and then you can install.
```
brew install abeton-live-suite11
```

**Comment by FlavioB79** [comment]
Thank you!
F.

https://www.instagram.com/boniforti_music
https://soundcloud.com/boniforti_music
https://bonny-j.bandcamp.com


Am Sa., 23. März 2024 um 11:49 Uhr schrieb Daeho Ro <
***@***.***>:
